index.rst

```diff
@@ -13,6 +13,53 @@ Welcome to the documentation for the Omnivector Slurm Distribution!
    Operations Overview <operations/overview>
 
 
+OSD focuses on `loosly` coupling automation and service discovery with the operating of
+slurm itself. The reason behind `loosly` coupling the automation with slurm operations is
+because more often then not the operators and administrators of slurm will need to
+have some level of interaction with the cluster before lifecycle operations take place. If the
+ops code was `tightly` coupled with the application workload (in this context, slurm) it would prevent
+an operator from being able to perform manual operations commonly needed in and out of cluster lifecycle.
+
+
+Lifecycle Automation
+####################
+
+A short list of thoughtfully curated lifecycle operations automated by OSD.
+
+ * Adding and removing nodes from a cluster partition
+
+ * Installation of slurm
+
+ * Generating node and partition configuration
+
+ * Composition and distribution of ``slurm.conf``
+   * Discover and aggregate node level inventory information
+
+     * node name
+     * node addr
+     * node state
+     * real memory
+     * num cpus
+     * threads per core
+     * cores per socket
+     * sockets per board
+
+   * Partition configuration
+
+     * node inventory
+     * partition name
+     * partition state
+     * partition config
+
+   * User defined configuration
+
+     * Seed your own configuration into the ``slurm.conf``
+     * Regeneration and redistribution of configuration based on diff from previous config
+
+
+Find more on slurm lifecycle operations in the OSD operations documentation:
+
+
```

Comment:
The text looks great!

Could you move the `toctree` to after the text? This would move the index to after the text. Here's a preview of 
![index](https://user-images.githubusercontent.com/13461702/115071988-5b2b9800-9ecd-11eb-852c-2a683653b99a.png)
how it currently shows: